enterprise/frontend/src/embedding-sdk/components/public/StaticQuestion/StaticQuestion.tsx

```diff
@@ -26,19 +30,26 @@ const StaticQuestionInner = ({
   className,
   style,
   initialSqlParameters,
+  withDownloads,
 }: StaticQuestionProps): JSX.Element | null => (
   <InteractiveQuestionProvider
     questionId={initialQuestionId}
     variant="static"
     initialSqlParameters={initialSqlParameters}
+    withDownloads={withDownloads}
   >
-    {withChartTypeSelector && <InteractiveQuestion.ChartTypeDropdown />}
-    <InteractiveQuestion.QuestionVisualization
-      height={height}
-      width={width}
-      className={className}
-      style={style}
-    />
+    <Stack gap="sm">
+      <Group justify="space-between">
+        {withChartTypeSelector && <InteractiveQuestion.ChartTypeDropdown />}
+        {withDownloads && <InteractiveQuestion.DownloadWidgetDropdown />}
+      </Group>
```

Comment:
If both `withChartTypeSelector` and `withDownloads` are disabled (default case), does that affect the layout (i.e. will the QuestionVisualization gets shown fully as before)? I guess it should be the same as before.